src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -242,35 +244,35 @@ class SaagieClient {
         }
     }
 
-    String createProjectJobWithOrWithFile(){
-        if (configuration.jobVersion?.packageInfo?.name != null) {
-            createProjectJobWithGraphQL()
+    String createProjectJobWithOrWithFile(Job job, JobVersion jobVersion){
```

Comment:
please rename `createProjectJobWithOrWithFile` to a more understandable name (I think you mean createProjectJobWithOrWithoutFile)